.github/issue_template.md

```diff
@@ -35,6 +36,9 @@ Welcome! Before creating a new issue:
 <!-- Results of `conda list jupyterlab-git` or `pip show jupyterlab-git` -->
 - Extension version:
 <!-- Results of `jupyter labextension list` -->
+- Git version:
+<!-- Results of `git --version` -->
+- OS:
```

Comment:
```suggestion
- Operating System and version:
```
This is more explicit and follows the issue template used by jupyterlab
![image](https://user-images.githubusercontent.com/10111092/84597471-317b8d80-ae32-11ea-8bdc-e7fb43106353.png)